main.go

```diff
@@ -23,41 +23,61 @@ var versionCommand = &cobra.Command{
 
 func init() {
 	rcommand.AddCommand(versionCommand)
-	rcommand.AddCommand(listCommand)
+	rcommand.AddCommand(countCommand)
+	countCommand.AddCommand(podCommand)
 	rcommand.AddCommand(deployCommand)
-	rcommand.PersistentFlags().StringVarP(&envVar.Host, "server", "s", "myurl", "this is the cluster url")
-	rcommand.PersistentFlags().StringVarP(&envVar.Bearertoken, "token", "t", "usertoken", "this is the user token")
-	rcommand.PersistentFlags().StringVarP(&envVar.Namespace, "namespace", "n", "", "this is the namespace")
+
+	deployCommand.Flags().StringVarP(&envVar.Host, "server", "s", "", "this is the cluster url")
```

Comment:
Why do we have code changes here?

Reply:
The reason to change code here, so that it should not show list of flags at root level. 
For example considering git command line utilities, when user run git, it does not list the flags and only shows available commands. I am not sure if this is the right approach and would like to discuss more.
![Screenshot from 2020-06-22 12-51-28](https://user-images.githubusercontent.com/43685848/85314185-1c80a900-b487-11ea-90bb-c28ffbd0fa69.png)

Reply:
@sbose78 Can you please share your thoughts here?

Reply:
@sbose, removed all chnages which were not related to https://github.com/cli-playground/kodo/issues/10